packages/ui-kit/src/utils/code-block-parse-options.ts

```diff
@@ -1,19 +1,25 @@
 import parseNumericRange from './parse-numeric-range';
 
-const getLinesForRange = (value) => {
+type OptionValue = string | boolean | number[] | undefined;
```

Comment:
This whole area is ugly and may be refactored completely later.  The purpose of the options parsing is that it changes the type and data of a given property from string to e.g. an array of numbers.  ideally the implentation would have separate types for the input and output but that's not how the code using it is build.  

To not overengineer I left this here.